src/Consultation.vue

```diff
@@ -6,15 +6,29 @@
       </tr>
       <tr>
         <p>-ご相談内容-</p>
-        <textarea rows="12" cols="50"></textarea>
+        <textarea rows="12" cols="50" v-model="consultation"></textarea>
       </tr>
     </table>
     <router-link to="/Question" tag="button">前へ戻る</router-link>
-    <router-link to="/" tag="button">次へ進む</router-link>
+    <router-link to="/result" tag="button" v-on:click.native="addConsultation()">次へ進む</router-link>
   </div>
 </template>
 
 <script>
+
+
+export default {
+  data(){
+    return{
+      consultation:''
+    }
+  },
+  methods:{
+    addConsultation(){
```

Comment:
`{}`の前後にはスペースを空けるのが通例ですので修正しましょう。
```suggestion
  data() {
    return {
      consultation:''
    }
  },
  methods: {
    addConsultation() {
```